Source/DataSources/GeoJsonDataSource.js

```diff
@@ -508,6 +525,7 @@ define([
      * @param {Color} [options.stroke=GeoJsonDataSource.stroke] The default color of polylines and polygon outlines.
      * @param {Number} [options.strokeWidth=GeoJsonDataSource.strokeWidth] The default width of polylines and polygon outlines.
      * @param {Color} [options.fill=GeoJsonDataSource.fill] The default color for polygon interiors.
+     * @param {Boolean} [options.clampToGround=GeoJsonDataSource.clampToGround] true if we want the features on the ground.
```

Comment:
Expand this to mention that `Entity.polyline` will be replaced by `Entity.corridor`, since existing code will break if users toggle it without knowing it.